### ISSUE TITLE
Lots of duplicate sms

### ISSUE THREAD

**Issue by NateJackman** [joaomgcd/JoinChrome#68]
I experience lots of duplicated sms and random ordered messages with the chrome plugin

**Comment by joaomgcd** [comment]
Thank you for the report. Do these duplicates not appear on your phone? Does it help if you disable and re-enable the SMS service on your phone?

**Comment by NateJackman** [comment]
no, only in the chrome extension,  re-enabling didn't make a difference.

**Comment by joaomgcd** [comment]
Thanks. Do you have another Android device by any chance?

**Comment by NateJackman** [comment]
Just the one

**Comment by joaomgcd** [comment]
Can you check if this is fixed on the latest Android version?
